toolkit/tools/pkg/imagecustomizerlib/imageutils.go

```diff
@@ -184,6 +191,10 @@ func createImageBoilerplate(imageConnection *ImageConnection, filename string, b
 	// options for us. If we wanted to handle this more directly, we could create a golang wrapper around libmount
 	// (which is what findmnt uses). But we are already using the findmnt in other places.
 	tmpFstabFile := filepath.Join(buildDir, chrootDirName+"_fstab")
+	err = file.RemoveFileIfExists(tmpFstabFile)
+	if err != nil {
```

Comment:
should we ignore this error?

Reply:
No. It already ignores the error for when the file doesn't exist. Any other errors will likely be problematic.